walk/cache/cache.go

```diff
@@ -56,28 +64,16 @@ func PathsBucket(tx *bolt.Tx) *bolt.Bucket {
 	return tx.Bucket([]byte("paths"))
 }
 
-func deleteAll(bucket *bolt.Bucket) error {
-	c := bucket.Cursor()
-	for k, v := c.First(); !(k == nil && v == nil); k, v = c.Next() {
-		if err := c.Delete(); err != nil {
-			return fmt.Errorf("failed to remove cache entry for key %s: %w", string(k), err)
-		}
+func Clear(root string) error {
+	// determine the db location
+	path, err := Path(root)
+	if err != nil {
+		return err
 	}
 
-	return nil
-}
-
-func Clear(db *bolt.DB) error {
-	err := db.Update(func(tx *bolt.Tx) error {
-		err := deleteAll(PathsBucket(tx))
-		if err != nil {
-			return fmt.Errorf("failed to clear cache: %w", err)
-		}
-
-		return nil
-	})
-	if err != nil {
-		return fmt.Errorf("failed to clear cache: %w", err)
+	// force-remove the db file
+	if err = os.Remove(path); !(err == nil || os.IsNotExist(err)) {
```

Comment:
Do we support Windows? Windows doesn't let you delete files if they're still open in another process, right?